packages/app/app/locales/en.json

```diff
@@ -191,6 +191,7 @@
     "user": "User:",
     "youtube": "Youtube",
     "yt-api-key": "Youtube API Key",
+    "invidious-url": "Invidious instance url",
```

Comment:
This is ok for now but I'm making a mental note to introduce per-plugin settings at some point and configure stuff like this in the plugins section

Reply:
sound great